src/program/lwaftr/virt/lwaftrctl

```diff
@@ -4,99 +4,90 @@
 #
 # The scripts enables several commands:
 #
-# * lwaftrctl snabbnfv start|stop. Starts or stops snabbnfv.
-# * lwaftrctl vm       start|stop. Starts or stops virtual machine
-# * lwaftrctl lwaftr   start|stop. Starts or stops lwaftr inside virtual machine.
+# * lwaftrctl start|stop snabbnfv   Starts or stops snabbnfv.
+# * lwaftrctl start|stop vm         Starts or stops virtual machine
```

Comment:
I guess it's a matter of preference, no standardization there. No big deal anyway.